### ISSUE TITLE
inquire segfaults

### ISSUE THREAD

**Issue by pklehre** [ocamllabs/ocaml-multicore#622]
The inquire library seems incompatible with multicore. When running the example `select.exe`, I get the following error message:

```
? What's your favorite movie? 
   1) Star Wars: The Rise of Skywalker
   2) Solo: A Star Wars Story
 » 3) Star Wars: The Last Jedi
   4) Rogue One: A Star Wars Story
   5) Star Wars: The Force AwakensFatal error: exception Unix.Unix_error(Unix.EINVAL, "tcsetattr", "")

```

https://github.com/tmattio/inquire

**Comment by abbysmal** [comment]
Hello,

Thank you for your report!

We just landed #621 (minutes ago!) and it appears it fixed your issue.

```
●●●● dune exec -- example/select.exe
? What's your favorite movie? 
   1) Star Wars: The Rise of Skywalker
   2) Solo: A Star Wars Story
 » 3) Star Wars: The Last Jedi
   4) Rogue One: A Star Wars Story
   5) Star Wars: The Force Awakens
```

Recompiling Multicore OCaml with the latest changes will likely fix your troubles.

**Comment by pklehre** [comment]
Great!
